sysbuild/CMakeLists.txt

```diff
@@ -533,6 +533,10 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
   if(SB_CONFIG_SUIT_BUILD_RECOVERY)
     set_config_bool(${DEFAULT_IMAGE} CONFIG_SUIT_MPI_APP_RECOVERY y)
     set_config_bool(${DEFAULT_IMAGE} CONFIG_SUIT_MPI_RAD_RECOVERY y)
+
+    if (SB_CONFIG_SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_LOCAL_3)
+      set_config_bool(${DEFAULT_IMAGE} CONFIG_SUIT_MPI_APP_LOCAL_3 y)
+    endif()
```

Comment:
should this also set it to n if not selected? Otherwise it will be sticky and remain on if someone changes it in menuconfig

Reply:
Good question, I think in general - no, as the user still might need to use `APP_LOCAL_3` for other purposes, so there still might be cases in which `SB_CONFIG_SUIT_RECOVERY_APPLICATION_IMAGE_MANIFEST_APP_LOCAL_3=n` but `CONFIG_SUIT_MPI_APP_LOCAL_3=y`

If you know some "softer" way of setting a `CONFIG` based on `SB_CONFIG` (so that it doesn't remain sticky) some advice would be very appreciated :)